timev.rb

```diff
@@ -223,48 +223,58 @@ def self.now(in: nil)
     Primitive.time_s_now(Primitive.arg!(:in))
   end
 
-  # _Time_
+  # Returns a new \Time object based on the given arguments.
+  #
+  # Required argument +time+ may be either of:
+  #
+  # - A \Time object, whose value is the basis for the returned time;
+  #   also influenced by optional keyword argument +in:+ (see below).
+  # - A numeric number of seconds (since the epoch) for the returned time,
+  #   in the range <tt>0..60</tt>.
```

Comment:
I don't think this range is true? It's the number of seconds since the epoch, so it could be (and probably is) much larger than 60. It could also be negative to count backwards from the epoch.